.github/workflows/rust.yml

```diff
@@ -50,7 +50,7 @@ jobs:
           key: cache-${{ runner.os }}-target-dir-cache-${{ hashFiles('cargo.lock') }}
 
       - name: Target dir clean
-        if: steps.target-dir-cache.outputs.cache-hit != 'true' || steps.target-clean-cache.outputs.cache-hit != 'true'
+        # if: steps.target-dir-cache.outputs.cache-hit != 'true' || steps.target-clean-cache.outputs.cache-hit != 'true'
```

Comment:
The two previous steps are now not needed anymore if you don't want to cache `target`

Reply:
I don't want to completely remove them right now, I plan to revisit the CI as a speparate ticket as it has many problems like building in release mode unnecessarily.